packages/affine/model/package.json

```diff
@@ -23,6 +23,7 @@
     "@blocksuite/global": "workspace:*",
     "@blocksuite/inline": "workspace:*",
     "@blocksuite/store": "workspace:*",
+    "@preact/signals-core": "^1.8.0",
```

Comment:
why adding this?

Reply:
It was originally used to listen for signals in the paragraph model, but it is no longer needed and will be deleted.